### DISCUSSION TITLE
AS3 docs and guides offline

### DISCUSSION THREAD

**Discussion by joao-cesar** [airsdk/Adobe-Runtime-Support#3393] [General]
Does anyone know if this is a bug or intentional?

And does Harman have the guides and other official legacy learning materials hosted somewhere else?

Thanks.

**Comment by marchbold** [comment]
Which documentation are you referring to? 

We have been attempting to move everything to the https://airsdk.dev/ site.

**Reply by itlancer** [reply]
Related request: https://github.com/airsdk/Adobe-Runtime-Support/issues/3103

**Comment by joao-cesar** [comment]
From Adobe.

The official AS3 docs:
https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/index.html

And these, for example:
https://help.adobe.com/en_US/as3/learn/index.html
https://help.adobe.com/en_US/air/build/index.html

**Reply by marchbold** [reply]
Ah, interesting, yeah I suppose it had to happen eventually. Adobe hasn't maintained or updated that documentation and it is out of date anyway. 

The latest reference is here: https://airsdk.dev/reference and we had ported a lot of that other documentation already.

**Reply by joao-cesar** [reply]
Yeah, cool, I'm aware that the docs are available under the airsdk.dev.

But it would also be nice to have the guides and all possible legacy learning materials that have been lost after Adobe started to redirect everything AS3 related to that generic Flash Player's EOL page.

**Comment by Adolio** [comment]
It seems to be back online 🙂

I hope they won't shut it down without a redirection...

**Reply by joao-cesar** [reply]
Yeah, it seems to be.

Let's hope it continue working now.
